nomos-da/kzgrs/src/common.rs

```diff
@@ -179,4 +182,55 @@ mod test {
             })
         ));
     }
+
+    #[test]
+    fn blst_error_conversions() {
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_SUCCESS)),
+            "Operation successful"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_BAD_ENCODING)),
+            "Bad encoding"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_POINT_NOT_ON_CURVE)),
+            "Point not on curve"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_POINT_NOT_IN_GROUP)),
+            "Point not in group"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_AGGR_TYPE_MISMATCH)),
+            "Aggregate type mismatch"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_VERIFY_FAIL)),
+            "Verification failed"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_PK_IS_INFINITY)),
+            "Public key is infinity"
+        );
+        assert_eq!(
+            format!("{}", BlstError(BLST_ERROR::BLST_BAD_SCALAR)),
+            "Bad scalar value"
+        );
+
+        assert_eq!(
+            format!("{}", KzgRsError::from(BLST_ERROR::BLST_SUCCESS)),
+            "BLST error: Operation successful"
+        );
+    }
```

Comment:
I'm wondering why do we test this. As we are using a wrapping library, shouldn't it be tested in the library? :thinking:

Reply:
Which library do you refer to  ? This test is for coverage increase of fmt::Display implementation. Test with interesting context would be more valuable, I agree.